src/node_worker.cc

```diff
@@ -460,13 +460,13 @@ void Worker::New(const FunctionCallbackInfo<Value>& args) {
       // The first argument is program name.
       invalid_args.erase(invalid_args.begin());
       if (errors.size() > 0 || invalid_args.size() > 0) {
-        v8::Local<v8::Value> value =
+        v8::Local<v8::Value> value2 =
             ToV8Value(env->context(),
                       errors.size() > 0 ? errors : invalid_args)
                 .ToLocalChecked();
         Local<String> key =
             FIXED_ONE_BYTE_STRING(env->isolate(), "invalidExecArgv");
-        args.This()->Set(env->context(), key, value).FromJust();
+        USE(args.This()->Set(env->context(), key, value2).FromJust());
```

Comment:
@refack Ditto here – are these changes related? If they are, can you add comments, so that people don’t start undoing them by accident?

If they are unrelated, can you drop them from this PR? (At least the `USE()` seems redundant?)

Reply:
I tend to clean warnings in the areas of code I'm churning anyway.
* `value` was recycled
* `.FromJust()` resault was ignored.

Reply:
For `value`: We’ve at least once decided not to do this, see https://github.com/nodejs/node/pull/25920. In particular, I think adding `2` to a value makes this more confusing. However, I would agree that changing the variable name to something more meaningful, e.g. `exec_argv_errors` or similar would be helpful.

For `.FromJust()`, if that gives a warning it should also give a warning in a lot of other places? We specifically use `.FromJust()` to avoid warnings…

Either way, I’d still prefer to not mix this up with an actual bug fix and to put this into separate PRs/commits.

Reply:
Renamed `value2` -> `error`